ikuzo/storage/x/gorm/organization.go

```diff
@@ -44,8 +44,8 @@ func (o *OrganizationStore) Delete(ctx context.Context, id domain.OrganizationID
 	return o.db.Delete(domain.Organization{}, "ID = ?", id).Error
 }
 
-func (o *OrganizationStore) Get(ctx context.Context, id domain.OrganizationID) (domain.Organization, error) {
-	var org domain.Organization
+func (o *OrganizationStore) Get(ctx context.Context, id domain.OrganizationID) (*domain.Organization, error) {
+	var org *domain.Organization
 
 	if err := o.db.Where("ID = ?", id).First(&org).Error; err != nil {
```

Comment:
Je hebt van org een variable pointer declaration gemaakt. Waardoor het argument in First een adres van een adres is. Gaat dat wel goed dan?